pkg/installer/utils.go

```diff
@@ -20,16 +20,24 @@ import (
 	"context"
 	"fmt"
 	"math/rand"
+	"os"
 	"path"
+	"reflect"
 	"time"
 
+	"github.com/fatih/color"
 	"github.com/minio/directpv/pkg/k8s"
 	corev1 "k8s.io/api/core/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-var seededRand = rand.New(rand.NewSource(time.Now().UnixNano()))
+var (
+	bold       = color.New(color.Bold).SprintFunc()
```

Comment:
You could use `color.HiWhiteString()`

Reply:
can the color attributes be added over this based on the usage?

Reply:
No. You would need to use `color.HiWhiteString()` directly instead of `bold` i.e. you don't need `bold`